bcgov/HOUS-permit-portal
# Feature/bphh 665/view scheduled published versions

## Description
- Adds template version screen
- Adds side bar to check template version and resume draft



## What type of PR is this? (check all applicable)

- [X ] 🍕 Feature

## Related Tickets & Documents
-https://hous-bssb.atlassian.net/browse/BPHH-665


## Steps to QA
- Navigate to Template Catalogue as super admin
- Click Versions, select resume draft to navigate to edit screen
- Click Publish and schedule a version through the publish modal
- Note the permit and activity type
- Navigate back to the catalogue screen
- Click Versions button for the the permit and activity type noted previously
- You should be able to view the scheduled versions
